### ISSUE TITLE
Support of tensorflow.keras.preprocessing.image.ImageDataGenerator.flow_from_directory in Ocra or tfpark

### ISSUE THREAD

**Issue by leonardozcm** [intel-analytics/analytics-zoo#3144]
Using Orca.
when generating image and label for `Estimator.fit`, it seems that `DirectoryIterator `yielding tuples of `(x, y)` is not supported, but for keras user it is a common way to realize complex image preprocessing.

here is my code for example:

```
# Rescale all images by 1./255 and apply image augmentation
train_datagen = keras.preprocessing.image.ImageDataGenerator(
                rescale=1./255)
# Flow training images in batches of 20 using train_datagen generator
train_generator = train_datagen.flow_from_directory(
                train_dir,  # Source directory for the training images
                target_size=(image_size, image_size),
                batch_size=batch_size,
                # Since we use binary_crossentropy loss, we need binary labels
                class_mode='binary')

est = Estimator.from_keras(keras_model=model)
est.fit(data=train_generator ,  ##can't init TFDataDataset2
            batch_size=batch_size,
            epochs=epochs,
            validation_data=validation_dataset
           )
```

**Comment by yangw1234** [comment]
python generator currently cannot be supported as there is no way to parallelize it.

**Comment by jason-dai** [comment]
> python generator currently cannot be supported as there is no way to parallelize it.

I think we should look for ways to support generator (through JEP or Ray)
